pkg/controller/migstorage/provider.go

```diff
@@ -0,0 +1,100 @@
+package migstorage
+
+import (
+	"github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	event2 "sigs.k8s.io/controller-runtime/pkg/event"
```

Comment:
Why `event2`? May be missing something but don't see `event` taken by an existing import.

Reply:
There must have already been a local variable named `event` and my IDE did that to resolve the shadowing.  Will fix.